src/Kongverge/Helpers/ConsoleHelper.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.IO;
+
+namespace Kongverge.Helpers
+{
+    public class ConsoleHelper
+    {
+        public static string ReadStdIn()

```

Comment:
This won't be mockable, so it'll have to change once unit tests are added.

Reply:
You should be able to take advantage of this class: https://github.com/natemcmaster/CommandLineUtils/blob/master/src/CommandLineUtils/IO/PhysicalConsole.cs#L13